epigraphhub/data/_config.py

```diff
@@ -27,3 +27,8 @@
 OWID_CSV_PATH = "/tmp/owid/releases"
 OWID_FILENAME = OWID_CSV_URL.split("/")[-1]
 OWID_HOST = "135.181.41.20"
+
+"""
```

Comment:
maybe just using the single line comment would be enough like: 

# SINAN data config

Reply:
@luabida is this fixed?